docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+
+See Table 1 on page 11 of the
+[https://arxiv.org/abs/1910.11333](Supplementry Information)
+document for a description and comparison between average error, Pauli error,
+and depolarization error.
+
+The incoherent error is the "unitarity" of the gate or cycle.
+This is defined as the decay rate per gate (or cycle) of the
+[Purity](https://en.wikipedia.org/wiki/Purity_(quantum_mechanics))
+when fit to an exponential curve.  This rate has been scaled to match the
+average error per Clifford gate (or per 2-qubit cycle).
+For more about purity benchmarking, see Section 6.3 of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf).
+
+The purity error can be interpreted as a measure of the incoherent error,
+such as those caused by stochastic processes such as relaxation.  The average
+error can be interpreted as containing both this incoherent error as well as
+the coherent error resulting from improper control or calibration of the device.
+
+Note that, due to statistical fluctuations, it is possible that the purity
+error can exceed the average error by small amounts.
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+**Note that the metric names below are subject to change without notice.**
+
+### P_00 readout error
+*   Metric key: single_qubit_p00_error
+*   Metric key: parallel_p00_error
+
+The p_00 is defined as the probability that the state is measured as |0⟩ after
+being prepared in the |0⟩ state.  The p_00 error is defined as one minus this
+result.
+
+There are several sources of error in this model.  This error is primarily
+composed of error in measurement (readout) of the qubit while in the ground
+state.  However, this probability also contains the error than the qubit was not
+reset into the |0⟩ ground state properly.  This is often called the SPAM (state
+preparation and measurement) error.
+
+The single qubit error is when the readout is measured in isolation (only one
+qubit is measured at a time), while the parallel error is taken for all qubits
+at the same time.
+
+### P_11 readout error
+*   Metric key: single_qubit_p11_error
+*   Metric key: parallel_p11_error
+
+The p_11 is defined as the probability that the state is measured as |1⟩ after
+being prepared in the |1⟩ state.  The p_11 error is defined as one minus this
+result.
+
+This is dominated by the error in measurement (readout) of the qubit, but it
+implicitly contains several different types of error.  Also possible is that the
+excited state |1⟩ was not prepared correctly or that the state decayed before
+measurement.  This error is generally expected to be higher than the P_00 error.
+
+The single qubit error is when the readout is measured in isolation (only one
+qubit is measured at a time), while the parallel error is taken for all qubits
+at the same time.
+
+### Readout separation error
```

Comment:
Is this value accounted for in p_00, p_11? The way I'm interpreting the relationship is
```
p_00 = P("measured as |0⟩" | "prepared in the |0⟩ state")
         = P("measured as |0⟩" | "appeared in |0> region of IQ plane")P("appeared in |0> region of IQ plane" |"prepared in the |0⟩ state")
          = F_s P("appeared in |0> region of IQ plane" |"prepared in the |0⟩ state")
```
Otherwise it seems like I'd need to propagate error proportional to (1 - F_s) for any analysis that consumes p_00/p_11

Reply:
Yes, for p_00 and p_11, we cannot tell why the error occurred (in and of itself), whether it was decay, measurement error, or statistical error.

This metric purely measures the error we would expect if both |1> and |0> are Gaussian distributions sampled without other error.  (Since Gaussians are infinite, there will always be some, likely small, overlap)

Added a sentence to clarify this.